src/Rules/Api/BcUncoveredInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Api;
+
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\ExtendedMethodReflection;
+use PHPStan\Reflection\FunctionReflection;
+use PHPStan\Reflection\ReflectionProvider;
+use PHPStan\Type\Type;
+
+final class BcUncoveredInterface
```

Comment:
A food for thought - in the future we could add a new annotation like `@api-interface-forbit-implements`. That would allow us not to maintain these lists and also BackwardCompatibilityCheck with each new interface.